docs/acceptance-criteria/uacs-rfc-19-random-beacon-v2.adoc

```diff
@@ -0,0 +1,467 @@
+  - isn’t the other relay entry request already pending? +
+  - has the requester provided the exact required fee? (if the fee is lower or
+  higher, the transaction should be reverted) +
+  - is there at least one active signing group?
+
+| Relay request
+| If a relay request is submitted before the previous relay request finished
+  being processed, the new request should be rejected (transaction should be
+  reverted).
+
+| Relay request
+| If the relay entry request is deemed valid, then: +
+  - the fee provided by the requester should be deposited in the maintenance
+  pool (with an intention to cover the reimbursements for DKG-related actions) +
+  - the current relay entry should be used by the beacon to select a signing
+  group from the set of active groups (by hashing the relay entry and applying
+  `modulo no_of_active_groups` operation on the result) +
+  - the random beacon contract should emit an event informing about the signing
+  group selection +
+  - the beacon should start the period of signature submission eligibility +
+  - the selected signing group should start off-chain signature generation (upon
+  noticing event sent by the beacon).
+
+| Relay request
+| If the relay entry request is deemed invalid, then: +
+  - the transaction should be reverted.
+
+| Relay request - signature generation
+| Once the active group is selected as a signing group, it is tasked with the
+  generation of a BLS threshold signature (based on the current relay entry and
+  the data provided by at least `blsThreshold=(group_size/2) + 1` members of the
+  signing group).
+
+| Relay request - relay entry submission
+| During the signature submission eligibility period only selected members from
+  the group should be eligible to successfully submit the new relay entry
+  result. +
+  - at the beginning (right after the challenge period passes) only one member
+  (with index `new_entry % group_size` should be eligible +
+  - after fixed amount of blocks another member (with index `(new_entry %
+  group_size ) + 1` should become eligible (previous operator remains eligible)
+  - next members should become eligible after each fixed amount of blocks.
+
+| Relay request - relay entry submission
+| Upon receiving the transaction submitting the relay entry, the random beacon
+  contract should determine validity of the transaction.
+
+| Relay request - relay entry submission
+| When determining validity of the relay entry submission, the following aspects
+  should be taken into account: +
+  - is the submitter eligible to submit? +
+  - hasn’t the hard timeout for the relay request be reached?
+
+| Relay request - relay entry submission
+| When requesting a relay entry, it should be possible to pass an optional
+  address parameter - this is the address of a contract implementing
+  `IRandomBeaconConsumer` interface.
+
+| Relay request - relay entry submission
+| If the transaction submitting the new relay entry gets approved, then: +
```

Comment:
We should also mention about exceeding soft timeout and bleeding. This is because we perform the actual slashing of the bled out value upon entry submission.